docs/design/multi-table-pipelines/README.md

```diff
@@ -116,6 +116,16 @@ Transit Table pass traffic to the next switch, by default drops unmatched packet
  
 ## Modes Change and Customer Flow migration
 
+To migrate to Multi-Table mode operator should do the following steps for each switch:
+
+* Enable MultiTable for the switch via NorthBound API see `PUT` `/v1/switches/{switch-id}/properties`.
+* Validate switch and verify that there are no rule discrepancies
+* Sync flow that goes through the switch one by one, starting from `default` flows
```

Comment:
You have mixed kilda-flows and OF-flows in one sentence. I believe no one outside the development team can understand what you have requested to do here.